src/tabular-api-surface.ts

```diff
@@ -736,12 +759,30 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
         err = null;
       }
 
+      const mutationErrors = Array.from(mutationErrorsByEntryIndex.values());
       if (mutationErrorsByEntryIndex.size !== 0) {
-        const mutationErrors = Array.from(mutationErrorsByEntryIndex.values());
         callback(new PartialFailureError(mutationErrors, err));
         return;
       }
-
+      if (err && timeoutExceeded) {
```

Comment:
Do we want to omit the error code is some cases where the timeout hasn't been exceeded as well?

Reply:
There are RPC level error code and entry level error code. When the deadline is passed, rpc level error code should be deadline exceeded and entry level error code should be whichever code it gets from the server.

Reply:
Note that actually [the test is expecting no error code at the grpc level](https://github.com/googleapis/cloud-bigtable-clients-test/blob/c72bf49142e8208b586d51b240f46860ae5d4b28/tests/mutaterows_test.go#L226). Does the test need to change?